.palantir/revapi.yml

```diff
@@ -273,6 +273,15 @@ acceptedBreaks:
     - code: "java.method.addedToInterface"
       new: "method java.util.List<org.apache.iceberg.StatisticsFile> org.apache.iceberg.Table::statisticsFiles()"
       justification: "new API method"
+    - code: "java.method.addedToInterface"
```

Comment:
@gaborkaszab I think you need to just re-do the changes on this file. Only `new: "method org.apache.iceberg.ManageSnapshots org.apache.iceberg.Transaction::manageSnapshots()"` should be added to `1.1.0`

Reply:
Yeah, I know. I missed to amend the changes to my patch after re-doing the revapi. But anyway thanks for checking! :)